src/client/views/index.html

```diff
@@ -18,17 +18,19 @@
     </div>
     <div class="main">
       <div class="search">
-        <div class="destination">
-          <label for="destination">Where:</label>
-          <input type="text" id="destination" placeholder="Enter location">
-        </div>
-        <div class="departure">
-          <label for="departure-date">When:</label>
-          <input type="date" id="departure-date" placeholder="yyyy/mm/dd">
-        </div>
-        <div>
-          <button id="get-info" type="submit">Get info</button>
-        </div>
+        <form id="main-form">
+          <div class="destination">
+            <label for="destination">Where:</label>
+            <input type="text" required id="destination" placeholder="Enter location" />
+          </div>
+          <div class="departure">
+            <label for="departure-date">When:</label>
+            <input type="date" required="required" id="departure-date" placeholder="yyyy/mm/dd">
```

Comment:
I think it's better to use the same form of `required` attribute, e.g. a bit above it's written without a value.
I've found this answer https://stackoverflow.com/a/6531862 and it seems that it's just for different standards, as we're using HTML it should be fine to use a short form like

```html
<input type="text" required />
```